src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -107,3 +143,21 @@ def get_helm_registry(profile, location):
     telemetry.set_exception(exception=str(response.json()), fault_type=consts.Get_HelmRegistery_Path_Fault_Type,
                             summary='Error while fetching helm chart registry path')
     raise CLIError("Error while fetching helm chart registry path: {}".format(str(response.json())))
+
+
+def get_flag_from_values_file(values_file):
+    isClientSecretATokenFlag = None
+    with open(values_file) as f:
+        try:
+            docs = yaml.safe_load(f)
+            if 'systemDefaultValues' not in docs:
+                raise CLIError("Incorrect values file format. Provide 'isClientSecretAToken' flag under 'systemDefaultValues' .yml file.")
+            else:
+                if 'isClientSecretAToken' not in docs['systemDefaultValues']:
```

Comment:
Is docs loaded as a dictionary?

Reply:
Yes

Reply:
If thats the case, use docs.get('systemDefaultValues') instead of docs['systemDefaultValues']. It will prevent unexpected error in case the field is absent.

Reply:
The above if checks the absence of that field.